### DISCUSSION TITLE
Please tell me about your favorite headphone or earphone!

### DISCUSSION THREAD

**Discussion by tyamk** [NAIST-SE/AttractingContributors#1106] [2024 Attraction Contributions]
Please let me share this link: https://github.com/tyamk/AttractingContributors_HeadPhones-EarPhones/tree/main

**Comment by tyamk** [comment]
Still need 5 people more. Please help me with this project!

**Comment by tyamk** [comment]
I will definitely get back to you!

**Comment by Canppchk** [comment]
I submitted a Galaxy Buds add-on in your list.
could you help me with my project too? Thanks!!
https://github.com/Canppchk/AttractingContributors-Ramen

**Comment by tyamk** [comment]
Thank you!
add Bishiya

**Comment by tyamk** [comment]
I'm done! Thank you for your help!
I can contribute to your project if you need more people.
Please write down something in this discussion or give me a pull request.

**Comment by oyewaleoyebode** [comment]
I use Soundcore. It's very efficient and good.

**Comment by tyamk** [comment]
Thank you! 
I checked it. That is helpful!
